File: java-service/src/main/java/com/slimblog/entity/Article.java
package com.slimblog.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import java.io.Serializable;
import java.time.LocalDateTime;
import lombok.Getter;
import lombok.Setter;

/**
 * <p>
 * 
 * </p>
 *
 * @author yzq
 * @since 2022-03-19
 */
@Getter
@Setter
public class Article implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "id", type = IdType.AUTO)
    private Integer id;

    private Integer typeId;

    private String title;

    private String articleContent;

    private String introduce;

    private LocalDateTime createTime;

    private Integer viewCount;

    private Integer partCount;

    private String articleContentHtml;

    private String introduceHtml;

    private Boolean isTop;

    private LocalDateTime updateTime;


}

File: kotlin-service/src/main/kotlin/com/slimblog/model/entity/Article.kt
package com.slimblog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import java.time.LocalDateTime

/**
 * @author yanni
 * @date time 2022/3/19 21:33
 * @modified By:
 *
 */
data class Article(
    @TableId("id", type = IdType.AUTO)
    var typeId: Int,
    var title: String,
    var articleContent: String,
    var introduce: String,
    var createTime: LocalDateTime,
    var updateTime: LocalDateTime,
    var viewCount: Int,
    var isTop: Int
) {
}